slices/publicsuffix.yaml

```diff
@@ -0,0 +1,6 @@
+package: publicsuffix
+
+slices:
+  data:
+    contents:
+      /usr/share/publicsuffix/*:
```

Comment:
this path seems to be specific to this pkg, so the glob is a bit safer to use here. Nonetheless, given that there are only 3 files within, it is safer to just list them explicitly :)